app/modules/intelligence/agents/chat_agents/system_agents/debug_agent.py

```
@@ -92,6 +96,43 @@ def _build_agent(self, ctx: Optional[ChatContext] = None) -> ChatAgent:
+                "list_watches",
+                "debug_list_launch_configs",
+                "debug_list_adapters",
+            ]
+            extra = self.tools_provider.get_tools(
+                debug_tool_names,
+                exclude_embedding_tools=exclude_embedding_tools,
+            )
+            got = {getattr(t, "name", None) for t in extra}
+            missing = [n for n in debug_tool_names if n not in got]
+            if missing:
+                logger.warning(
```

Ruff (0.15.12)

[error] Too many arguments for `logging` format string

(PLE1205)